src/Sylius/Bundle/CoreBundle/Tests/Fixture/LocaleFixtureTest.php

```diff
@@ -39,6 +39,28 @@ public function locales_can_be_set(): void
         $this->assertConfigurationIsValid([['locales' => ['en_US' => true, 'pl_PL' => false, 'es_ES' => true]]], 'locales');
     }
 
+    /**
+     * @test
+     */
+    public function default_locale_is_always_added() : void
+    {
+        $this->assertProcessedConfigurationEquals(
+            [['locales' => ['en_US' => true, 'pl_PL' => false, 'es_ES' => true]]],
```

Comment:
Looks like this configuration test has been broken since #7346 because the fixture itself expect locales to be values, not keys.